internal/gensample/gensample.go

```diff
@@ -56,41 +61,19 @@ func InitGen(desc []*descriptor.FileDescriptorProto, sampleFnames []string, gapi
 	}
 
 	var wg sync.WaitGroup
-	errChan := make(chan error)
+	errChan := make(chan error, 2)
```

Comment:
Uhm, where do you read from `errChan`?  It seems like you're always returning `err==nil` below

Reply:
Duh I was dumb! Added.